Src/IronPython/Runtime/Operations/StringOps.cs

```diff
@@ -2604,67 +2597,31 @@ private class UnicodeEscapeEncoding : Encoding {
             }
         }
 
-        private class ExceptionFallBack : DecoderFallback {
-            internal ExceptionFallbackBuffer buffer;
-
-            #region check for Utf8 Fallback issue
-
-            private static bool isUtf8Bugged;
-
-            private class TestUtf8DecoderFallBack : DecoderFallback {
-                public override int MaxCharCount => 0;
-
-                public override DecoderFallbackBuffer CreateFallbackBuffer() => new TestUtf8DecoderFallbackBuffer();
-            }
-
-            private class TestUtf8DecoderFallbackBuffer : DecoderFallbackBuffer {
-                public override int Remaining => 0;
-
-                public override bool Fallback(byte[] bytesUnknown, int index) {
-                    if (index < 0) throw new Exception();
-                    return false;
-                }
-
-                public override char GetNextChar() => (char)0;
+        // This is an equivalent of System.Text.DecoderExceptionFallback
+        // except for the custom error message and a workaround for a UTF-8 bug.
+        // It is **not** suitable for use if the following Encoding methods are being employed:
```

Comment:
typo: use in

Reply:
sorry, I fail to see the typo...

Reply:
You're right, not sure what I was thinking.